### DISCUSSION TITLE
Streams support

### DISCUSSION THREAD

**Discussion by illian64** [zalopay-oss/jmeter-grpc-request#54] [General]
Hello! Are there any plans about support bidirectional streams in plugin? In TODO list (readme.md) I don't see any mentions about it.

**Comment by Amoratinos** [comment]
I think this would be a nice feature to have. At this moment I don't find too many testing tools supporting bidi streaming. I'll be glad to contribute to add this feature but not an expert in jMeter plugin development though. 

I checked that  `ClientCaller` class already has support to invoke Bidi streaming methods so I suppose that the challenges are in how to operate with jMeter GUI no?
